utils/compare/read_files.py

```diff
@@ -0,0 +1,104 @@
+import graph
+import json 
+import os
+from pathlib import Path
+import sys
+import csv
+
+def read_config_file(file_name):
+    with open(os.path.dirname(os.path.realpath(__file__))+ "/" + file_name, "r") as f: 
+        read_str = f.read()
+        return json.loads(read_str)
+
+
+def write_uuid_to_csv(filepath):
+    uuid_data = [["current uuid",uuid]]
+    
+    print('ids ' + str(ids))
+    id_data = ids
+    id_data.insert(0, "comparison uuids")
+    print('id dat' + str(id_data))
+    uuid_data.append(id_data)
+    print('uuid dat ' + str(uuid_data))
+    with open(filepath, 'w') as f:
+     
+        # using csv.writer method from CSV package
+        write = csv.writer(f)
+        write.writerows(uuid_data)
+
+data_func = "mean"
+previous_version = os.getenv("previous_version", False)
+tolerancy_num = os.getenv("tolerancy", -20)
+
+
+# burner filter - keys + metric of interest
+uuid=os.getenv("UUID").strip()
+meta = graph.getMetadata(uuid)
+baseline_uuids=os.getenv("BASELINE_UUID", "")
+
+# can add as many config files here as data points collected
+if meta["benchmark"] == "k8s-netperf" :
+    index = "k8s-netperf"
+    k8s_json = read_config_file("configs/k8s.json")
+    metrics = [k8s_json]
+elif meta["benchmark"] == "ingress-perf" :
+    index = "ingress-performance"
+    ingress_json = read_config_file("configs/ingress.json")
+    metrics = [ingress_json]
+else:
+    index = "ripsaw-kube-burner"
+
+    latency_json = read_config_file("configs/podlatency.json")
+    crio_json = read_config_file("configs/crio.json")
+    kubelet_json = read_config_file("configs/kubelet.json")
+    etcd_json = read_config_file("configs/etcd.json")
+    master_json = read_config_file("configs/master_node.json")
+    worker_agg_json = read_config_file("configs/worker_node.json")
```

Comment:
Did we miss on adding/removing this from the metrics list?

Reply:
Yes! Think I updated now